data/en/mbs_0123.json

```diff
@@ -0,0 +1,352 @@
+                                            "title": "‘Change in level of business activity’",
+                                            "description": "\"We gained a new two year overseas contract, which increased our exports this month.\""
+                                        },
+                                        {
+                                            "title": "‘Maintenance/shutdowns’",
+                                            "description": "\"Our activity is significantly lower this month as we had a two week planned maintenance shutdown.\""
+                                        },
+                                        {
+                                            "title": "‘Special/calendar events’",
+                                            "description": "\"We manufacture school uniforms so the increase this month is a typical seasonal peak for our business.\""
+                                        },
+                                        {
+                                            "title": "‘Weather’",
+                                            "description": "\"Bad weather caused a flood at one of our sites, this reduced the amount of work we could undertake and affected sales.\""
+                                        },
+                                        {
+                                            "title": "‘Price effects’",
+                                            "description": "\"We have increased the prices of our products/services. We do this every January, hence our sales figures appear higher.\""
+                                        },
+                                        {
+                                            "title": "‘Currency effects (increase/decrease in the currency value)’",
+                                            "description": "\"The depreciation of sterling has resulted in more overseas sales.\""
+                                        }
+                                    ]
+                                },
+                                "type": "TextArea",
+                                "mandatory": true,
+                                "label": "Comments",
+                                "id": "changes-answer-3",
+                                "q_code": "146h"
+                            }],
+                            "description": "<p>We rely on your commentary to ‘tell the story’ behind changes in figures.  By commenting here it will reduce the need for us to call you.</p>"
+                        }]
+                    }
+                ]
+            }]
+        },
+        {
+            "id": "summary-section",
+            "title": "Summary",
+            "groups": [{
+                "id": "summary-group",
+                "title": "Summary",
+                "blocks": [{
+                    "type": "Summary",
+                    "id": "summary"
+                }]
+            }]
+        }
+    ]
+}
```

Comment:
Happy with the schema apart from front page

Reply:
@shahi645 If your referring to 'Commission and fees' that should not be there anymore anyway, spec needs to be updated and same on the other survey as well

Reply:
Is this for definite?